.changelog/2731.txt

```diff
@@ -0,0 +1,23 @@
+```release-note:bug
+resource/mongodbatlas_project: Fixes resource when user doesn't have project owner permission
```

Comment:
```suggestion
resource/mongodbatlas_project: Avoids error when user doesn't have project owner permission
```
mentioning resource would redundant, also applies to data source entries

Reply:
thanks, changed in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2731/commits/9dcf7ff8113db6121602ab88715da2fb8b4a58bb